services/identity/TotpGenerator.ts

```diff
@@ -8,7 +8,7 @@ interface TotpGeneratorProps {
 class TotpGenerator {
   generator: typeof totp
 
-  expiry: number
+  expiry?: number
```

Comment:
Why do we make this optional? Could we enforce that the `OTP_EXPIRY` env var is set instead?

Reply:
We don't enforce because the original implementation defaults it to 5! I'm fine either way but I chose to kept it close to the original!

Reply:
wait no u r right, i made a mistake - this would never be `undefined` as we default it in the constructor... will change